client/post-editor/editor-term-selector/index.jsx

```diff
@@ -82,6 +84,7 @@ export default connect(
 		return {
 			postType: getEditedPostValue( state, siteId, getEditorPostId( state ), 'type' ),
 			postTerms: getEditedPostValue( state, siteId, postId, 'terms' ),
+			canEditTerms: canCurrentUser( state, siteId, 'manage_categories' ),
```

Comment:
>Wouldn't it just be best to get `edit_terms` added into the API response and refactor then?

The value for `taxonomyObject.capabilities.edit_terms` can be any [valid capability](https://codex.wordpress.org/Roles_and_Capabilities#Capabilities), which is hard to accommodate in our current approach for capabilities on the `/sites/%s` endpoint.

The idea that I was referencing was that we could check to see if the value for the capability is one that we're aware of (i.e. exists on the site object). If so, we'd check that capability, otherwise fall back to some default (like the one here).